src/features/import/utils/prepareImportOperations.spec.ts

```diff
@@ -981,4 +981,104 @@ describe('prepareImportOperations()', () => {
       },
     ]);
   });
+
+  it('correctly parses and converts phone numbers with obvious formatting errors', () => {
+    const configData: Sheet = {
+      columns: [
+        {
+          field: 'phone',
+          kind: ColumnKind.FIELD,
+          selected: true,
+        },
+        { idField: 'id', kind: ColumnKind.ID_FIELD, selected: true },
+      ],
+      firstRowIsHeaders: false,
+      rows: [
+        {
+          data: ['+46732789887', 1],
+        },
+        {
+          data: ['46732789887', 2],
+        },
+        {
+          data: ['0046732789887', 3],
+        },
+        {
+          data: ['-460732789887', 4],
+        },
+        {
+          data: ['46732-78-98-87', 5],
+        },
+        {
+          data: ['460732789887', 6],
+        },
+        {
+          data: ['0732789887', 7],
+        },
+        {
+          data: ['+46 73278 98 87‬', 8],
+        },
```

Comment:
Also here, a comment explaining why this data is not valid would be amazing (i am getting a warning from gitHub on this line so I assume there are some "weird characters" in this one.

Reply:
Done